docs/DeveloperGuide.md

```diff
@@ -213,9 +213,45 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
-### \[Proposed\] Data archiving
 
-_{Explain here how the data archiving feature will be implemented}_
+### \[Proposed\] Deck System
+This feature would allow the user to create multiple lists of entries rather than having 
+all translations together in the same list. This means that the user could have different decks
+for different languages or even multiple decks for the same languages. 
```

Comment:
I think maybe we could provide written examples of what kind of different decks users can have, to help them understand better
For e.g. 
- Spanish
- Japanese

OR

- Spanish Animals
- Spanish Food

Reply:
Done